pandas/core/dtypes/cast.py

```diff
@@ -1422,6 +1422,8 @@ def convert_dtypes(
         if is_string_dtype(inferred_dtype):
             if not convert_string:
                 return input_array.dtype
+            elif inferred_dtype is "bytes":
```

Comment:
This is a memory comparison not a value comparison - running it would give
`<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?`

Reply:
Done